ui/src/app/applications/components/application-details/application-details.scss

```diff
@@ -258,19 +260,21 @@ $header: 120px;
             padding-bottom: 6px;
         }
 
-        .zoom-value {
+        .panel-options-text {
             user-select: none;
             margin-top: 3px;
             margin-right: 6px;
             margin-left: 4px;
             font-size: 14px;
             text-align-last: right;
-            float: right;
             width: 40px;
             border: 1px $argo-color-gray-5 solid;
             background-color: $argo-color-gray-3;
             padding: 2px;
             color: $argo-color-gray-7;
+            &.pod-group-count {
```

Comment:
This is unused? (From when you made the group count customizable?). This probably can be removed.

Reply:
You are right this CSS is left over from group count customizable.  I'll the removed the code